_posts/platform/app/2000-01-01-run-scheduled-tasks.md

```diff
@@ -47,3 +58,35 @@ scalingo --app my-app cron-tasks
 | */10 * * * * echo biniou |      |
 +--------------------------+------+
 ```
+
+## Delete Configured Cron Tasks
+
+In order to delete a cron tasks, you must delete the associated entry in the `cron.json`.
+If there is no more tasks, or if you want to remove all tasks, you can also remove the file.
+
+The cron tasks configuration will be updated when redeploying the application.
+
+## FAQ
+
+### How To Enable Cron Tasks?
+
+The cron feature is enabled by default.
```

Comment:
Nope. You need to ask the support. It is not globally available yet. You should say that it is currently in beta state but it is possible to get access to it by asking the support.